pages/common/virsh-pool-build.md

```diff
@@ -0,0 +1,9 @@
+# virsh pool-build
+
+> Build the underlying storage system for a virtual machine storage pool as defined in it's configuration file (create a configuration file using `virsh pool-define-as`).
+> See also: `virsh`.
+> More information: <https://manned.org/virsh>.
+
+- Build the storage pool specified by name or UUID (determine the name or UUID using `virsh pool-list` and start the pool once built using `virsh pool-start`):
```

Comment:
I'd say that this is implied.  How about
```suggestion
- Build the storage pool specified by name or UUID (determine using `virsh pool-list` and start the pool once built using `virsh pool-start`):
```

Reply:
It seems like the brackets are just indicating how it works under the hood which doesn't really suit the definition of "too long; didn't read" (the phrase, not the repo).
Could something like this be better? Users just need to know what it does, not how it works.

```suggestion
- Build and start the storage pool specified by name or UUID:
```

Reply:
My thinking is that the commands need context to be used effectively. If someone doesn't know how to determine the name or UUID, the example isn't going to be very useful. Similarly, what if someone doesn't know what to do with a pool once it's built?

Reply:
But that's why we have other examples or "See also" pages, right?

For example, in commands like `snap`, `apt`, `pacman` etc, we don't document how to get a list of packages when removing a package.

I do concede to your point in terms of convenience, however. It'd certainly be easier to have all the information in one place. I just also think documented 3 commands in a single tldr-ed command might be iffy.

If others agree, I'm all for it.

Reply:
Sorry it's taken me so long to respond @SethFalco. 

How would you feel if I moved just the references to the other tldr-pages to the "See also" line and omitted the context? E.g.:

```
> See also: `virsh`, `virsh-pool-list`,  `virsh-pool-start`
```

I prefer it the way it is because it doesn't put the onus on the reader try to figure out which "See also" applies to which example. Another option in keeping with this would be to change the reference to be an inline parenthetical "See also":

```
- Build the storage pool specified by name or UUID (See also: `virsh-pool-list`,  `virsh-pool-start`):
```

Reply:
I _subjectively_ prefer the first example in your response, where a command only documents the single command it's for and not dependencies.

It seems that it's 2 against 1 here, so I'll back down.
@bl-ue can have the final say on this, with the conversation presented above. What do you think?

Reply:
I prefer the first example of @aherst's response, simply because it looks odd to have a "See also" in an example description.

Reply:
Understood, then I concede.  
@aherst You can just scrap everything I said then. 😆 Sorry for the noise!

It's best to just heed bl-ue original advice.

Reply:
Not noise at all! Feedback is always appreciated. You've made me think about which references are best to put in the "See also" line and which are best to put inline as parenthetical comments. I'm going to make some changes as a result.

Reply:
@SethFalco, @bl-ue: I've made some changes based on your feedback. I've moved everything that didn't provide information needed for a specific example to the "See also" line. I think this reads better and still provides readers with the information they are in immediate need for.